docs/competition/event/incident-management.md

```diff
@@ -0,0 +1,85 @@
+
+Examples might include;
+
+* Accident requiring significant first aid,
+* Battery failure renders access to an area unsafe
+
+As with minor incidents, the initial reaction should be to notify the relevant coordinator and follow their instructions. If first aid is required the First Aid Provider should __always__ be notified first.
+
+## Stopping the competition
+
+A major inicident may call for the competition to be stopped. This may be at the end of a current match or in the middle of a match.
+
+Due to the impact of stopping the competition, ability to do so is restricted to the following roles:
+
+* Health and Safety Cooordinator,
+* Event Coordinator,
+* Head Judge,
+* Majority decision from Trustees.
+
+Stopping the competition will be indicated by a call over the radio of "This is _role_; close the arena."
+
+At this point the following should occur;
+
+* Matches stop,
+* Power tools area to be vacated,
+* Battery charging to be turned off,
+* Competitors, Team Leaders, and members of the public should be asked to remain in their current location,
+* Competitors to be asked to stop testing on robots.
+
+The aim of these is to prevent any other issues occuring whilst the current incident is being dealt with, and to prevent attendees from getting in the way of the handling the incident.
+
+## Shift in roles
+
+In the event of a major incident, the Head Shepherd becomes the point of contact between the main radio network and the shepherd radio network. They are also responsible for managing the Shepherds and Badgers. To avoid confusion, all radios should remain on their normal network.
+
+Shepherds become responsible for managing competitors, team leaders, and members of the public.
+
+The nearest roving helper should shutdown the power tools area, and remain there to ensure it remains unused.
+
+All other roles should remain in location and await further instructions.
+
+Further instructions will be given when appropriate, and should not be requested.
+
+## Stopping matches
+With stopping matches, the Head Judge should make a decision about whether to stop the match part way through, or to stop the matches once the current match is finished. This decision should be based on the proximity of the incident to the arena, and any further information received regarding the incident.
+
+If the match is stopped at the end, the Marshalls should score the match as normal, and then hit the power cut off on each robot. The arena should then be vacated and left as-is.
+
+If the match is stopped immediately, if safe to do so, the Marshalls should hit the power cut off on each robot, and then vacate the arena.
+
+In either scenario; the Badgers should ask all competitors to leave the staging area. Robots to remain where they are, power cut offs to be hit.
+
+## Resuming/cancelling the competition
+
+Resuming the competition requires the consent of all involved in stopping it.
+
+Cancelling the event requires a majority vote from Trustees.
```

Comment:
Should there be a fallback here in case the trustees are unavailable?

I _think_ all the trustees will be at this year's event, but I'm not sure that will always be the case.

Reply:
Probably. Perhaps it should be limited to trustees that are present, and extended to include the roles mentioned above?

Reply:
Yeah, or maybe trustees first and then, if there aren't a quorate number present then encourage them to consult with the above roles?